src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,23 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
```

Comment:
`post.user` might be `null` (`getUserBy` can return `null`) - in this case you shouldn't render UserInfo component to avoid errors (and it doesn't make sense w/o data in `post.user`)